data/datacatalogue/molgenis.csv

```diff
@@ -1,5 +1,5 @@
 tableName,tableExtends,columnName,columnType,key,required,refSchema,refTable,refLink,refBack,validation,semantics,description
-Version,,,,,,,,,,,,2.7
+Version,,,,,,,,,,,,3
```

Comment:
Breaking change is due to deleting of Contributions table and moving of some data items (resource, contributionType) in that table to Contacts table. I think it would be possible to write a migration for that.